packages/vhd-lib/src/Vhd/VhdAbstract.js

```diff
@@ -221,6 +222,12 @@ export class VhdAbstract {
     const aliasDir = path.dirname(path.resolve('/', aliasPath))
     // only store the relative path from alias to target
     const relativePathToTarget = path.relative(aliasDir, path.resolve('/', targetPath))
+
+    if (relativePathToTarget.length > ALIAS_MAX_PATH_LENGTH) {
+      throw new Error(
+        `Alias relative path ${relativePathToTarget} is too long : ${relativePathToTarget.length} chars, max is 1024`
```

Comment:
```suggestion
        `Alias relative path ${relativePathToTarget} is too long : ${relativePathToTarget.length} chars, max is ${ALIAS_MAX_PATH_LENGTH}`
```

Reply:
That's why I shouldn't have used a magic value :smile:

Reply:
Haha, yep.